src/Read.cpp

```diff
@@ -74,7 +74,7 @@ void Read::initialize() {
 }
 
 bool Read::readFile(const std::string& filePath, const std::string& pointCloudFrameId) {
-  if (filePath.find(".ply") != std::string::npos) {
+  if (filePath.substr(filePath.find_last_of(".") + 1) == "ply") {
```

Comment:
Hello @remod!

I think you have a good point, I have pushed the commit with std::filesystem::path::extension instead of the approach I used initially.